### ISSUE TITLE
Coolify Compatible Trigger.dev Template for Worker setup

### ISSUE THREAD

**Issue by Haji-sudo** [Mortalife/trigger-dev-coolify#2]
Hi, thanks for this complete repo, it made my day

Is it possible to create a docker-compose to build just the worker and explain how to connect it to the webapp?

I had a scenario where multiple servers were connected to a webapp as workers, is this possible?

**Comment by Mortalife** [comment]
I don't believe multiple works are possible currently. You can only have a single worker, the ability to break it out is to allow you to scale vertically the worker independently of the rest of the platform.

This is coming with the V2 Engine https://trigger.dev/launchweek/0/run-engine-2-alpha

There's an example of which services need to run on a different server in their own self-hosted.

**Comment by matt-aitken** [comment]
That's correct. In the current version we only support a single entity pulling from the queue, for the Docker setup that limits it to a single server. Our cloud setup is a lot more complicated and not a good fit for self-hosting.

When we ship v2 of the Run Engine you'll be able to add as many worker servers as you want and each will pull runs from the queue as long as a couple of env vars are set.
